libr/util/file.c

```diff
@@ -448,12 +448,12 @@ R_API char *r_file_slurp_random_line_count(const char *file, int *line) {
 	int start = *line;
 	if ((str = r_file_slurp (file, &sz))) {
 		gettimeofday (&tv, NULL);
-		srand (getpid() + tv.tv_usec);
+		r_srand (getpid() + tv.tv_usec);
```

Comment:
using getpid is probably a bad idea, mainly because the logic of such seed initialization should be in r_rand, not in here.